### ISSUE TITLE
Stop forcing widget title

### ISSUE THREAD

**Issue by flowdee** [flowdee/tomparisde-twitchtv-widget#16]
Change:
```
$title = ( ! empty( $instance['title'] ) ) ? $instance['title'] :
esc_html__( 'New title', 'tomparisde-twitchtv-widget' );
```

to:
`$title = ( ! empty( $instance['title'] ) ) ? $instance['title'] : '';`

**Comment by flowdee** [comment]
- https://secure.helpscout.net/conversation/1003034402/17438/

**Comment by flowdee** [comment]
Available in v2.0.5
